pnpm-lock.yaml

```diff
@@ -0,0 +1,14246 @@
+lockfileVersion: '9.0'
+
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
+
+importers:
+
+  .:
+    dependencies:
+      '@babel/compat-data':
+        specifier: ^7.21.5
+        version: 7.24.4
+      '@bcrs-shared-components/breadcrumb':
+        specifier: 2.1.24
+        version: 2.1.24
+      '@bcrs-shared-components/corp-type-module':
+        specifier: 1.0.15
+        version: 1.0.15
+      '@bcrs-shared-components/enums':
+        specifier: 1.1.7
+        version: 1.1.7
+      '@bcrs-shared-components/folio-number-input':
+        specifier: ^1.1.44
+        version: 1.1.44
+      '@bcrs-shared-components/genesys-web-message':
+        specifier: 1.0.0
+        version: 1.0.0(vue-class-component@7.2.6(vue@2.7.10))(vue@2.7.10)
+      '@bcrs-shared-components/interfaces':
+        specifier: 1.0.67
+        version: 1.0.67
+      '@bcrs-shared-components/staff-payment':
+        specifier: 1.0.29
+        version: 1.0.29(vue@2.7.10)
+      '@bcrs-shared-components/web-chat':
+        specifier: 2.0.0
+        version: 2.0.0(vue@2.7.10)
+      '@mdi/font':
+        specifier: ^4.5.95
+        version: 4.9.95
+      '@sentry/browser':
+        specifier: ^7.49.0
+        version: 7.110.1
+      '@sentry/integrations':
+        specifier: ^7.49.0
+        version: 7.110.1
+      axios:
+        specifier: ^0.27.2
+        version: 0.27.2
+      babel-plugin-transform-decorators:
+        specifier: ^6.24.1
+        version: 6.24.1
+      core-js:
+        specifier: ^3.30.1
+        version: 3.37.0
+      http-status-codes:
+        specifier: ^1.4.0
+        version: 1.4.0
+      launchdarkly-js-client-sdk:
+        specifier: ^2.24.2
+        version: 2.24.2
+      lodash:
+        specifier: ^4.17.21
+        version: 4.17.21
+      qs:
+        specifier: ^6.12.1
+        version: 6.12.1
+      quill:
+        specifier: ^2.0.0
+        version: 2.0.0
+      remove-accents:
+        specifier: ^0.4.2
+        version: 0.4.4
+      sbc-common-components:
+        specifier: 3.0.8
+        version: 3.0.8
```

Comment:
For some reason, this is out of date. (The package file specifies 3.0.12.)

Anyway, I updated this in PR #764.